docker/icra2023_tutorial/Dockerfile

```diff
@@ -136,18 +136,35 @@ WORKDIR /home/$USERNAME
 RUN sudo rosdep init && rosdep update
 
 # Add colcon mixins
-RUN colcon mixin add default https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml && colcon mixin update default && rm -rf log
-
-# Build and install the ROS 2 documentation
-RUN cd /home/$USERNAME && git clone https://github.com/ros2/ros2_documentation && cd ros2_documentation \
- && pip3 install -r requirements.txt && PATH=$PATH:/home/$USERNAME/.local/bin make multiversion \
- && mkdir /home/$USERNAME/docs.ros.org \
- && cp -av /home/$USERNAME/ros2_documentation/build/html/* /home/$USERNAME/docs.ros.org \
- && rm -rf /home/$USERNAME/ros2_documentation
-
-RUN mkdir -p /home/$USERNAME/ws_moveit2/src && cd /home/$USERNAME/ws_moveit2/src && git clone https://github.com/ros-planning/moveit2_tutorials -b humble --depth 1 && git clone https://github.com/ros-planning/moveit_task_constructor.git -b humble --depth 1
-
-# Copy Gazebo Fuel data from host if it exists
+RUN colcon mixin add default https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml \
+ && colcon mixin update default \
+ && rm -rf log
+
+# Optional: Build and install the ROS 2 documentation
+# FIXME new Sphinx version error. Temporarily comment out
+#RUN cd /home/$USERNAME && git clone https://github.com/ros2/ros2_documentation && cd ros2_documentation \
+# && pip3 install -r requirements.txt && PATH=$PATH:/home/$USERNAME/.local/bin make multiversion \
+# && mkdir /home/$USERNAME/docs.ros.org \
+# && cp -av /home/$USERNAME/ros2_documentation/build/html/* /home/$USERNAME/docs.ros.org \
+# && rm -rf /home/$USERNAME/ros2_documentation
```

Comment:
This is fixed by #44, so uncomment this and rebase on top of that once it is merged.

Reply:
Done in 79a6041